FreeAPS/Sources/Modules/DataTable/View/DataTableRootView.swift

```diff
@@ -315,7 +315,7 @@ extension DataTable {
                                     value: $state.externalInsulinAmount,
                                     formatter: insulinFormatter,
                                     autofocus: true,
-                                    cleanInput: true
+                                    liveEditing: true
```

Comment:
here and in other places `cleanInput: true` is the default, so I removed it, and added `liveEditing: true`

in the diff it looks like just a change of the parameter name